docs/supportedsites.md

```diff
@@ -289,6 +289,7 @@
  - **IPrima**
  - **iqiyi**: 爱奇艺
  - **Ir90Tv**
+ - **iTunes**
```

Comment:
This file is automatically updated during a release. Don't touch it in normal commits.